.github/workflows/webhook.yml

```diff
@@ -0,0 +1,17 @@
+name: Discord Automation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  commit-stream:
+    runs-on: ubuntu-latest
+    steps:
+    - env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+      uses: Ilshidur/action-discord@0.3.2
+      if: "${{ env.DISCORD_WEBHOOK != '' }}"
+      with:
+        args: '{{ EVENT_PAYLOAD.head_commit.message }} committed by {{ EVENT_PAYLOAD.head_commit.author.name }}'
```

Comment:
The previous format of "author before commit" is more natural to read. My previous comment was just calling out the possible ambiguity involved when parsing the messages produced by this job; This format also has ambiguity, since ` committed by ` is not a reserved construct that is unable to appear in the commit message.